src/wavesurfer.js

```diff
@@ -115,6 +115,8 @@ import MediaElementWebAudio from './mediaelement-webaudio';
  * @property {boolean} splitChannels=false Render with separate waveforms for
  * the channels of the audio
  * @property {SplitChannelOptions} splitChannelsOptions={} Options for splitChannel rendering
+ * @property {boolean} vertical=false Render the waveform vertically instead of horizontally.
+ * This may not be compatible with any plugins besides the regions plugin.
```

Comment:
i would move that statement to changelog instead and add 'currently'

Reply:
and build is failing on ci

Reply:
I don't see it. Where should I be looking?
<img width="867" alt="Screen Shot 2021-03-14 at 2 58 53 PM" src="https://user-images.githubusercontent.com/343665/111085729-cd2b4e80-84d5-11eb-9973-a77b11e9db4f.png">